roles/openshift_control_plane/templates/master.yaml.v1.j2

```diff
@@ -127,6 +127,12 @@ networkConfig:
   clusterNetworks:
   - cidr: {{ openshift_cluster_network_cidr }}
     hostSubnetLength: {{ openshift_host_subnet_length }}
+{% for cluster_network in openshift_cluster_networks %}
+{%   if cluster_network.cidr != openshift_cluster_network_cidr %}
```

Comment:
nit: for consistency with the rest of the file, will be nice to remove the extra space

`{% if cluster_network.cidr != openshift_cluster_network_cidr %}`

Reply:
This is the only place in the file where an `{% if ... %}` occurs inside of a `{% for ... %}`. The extra indent is there to highlight this nesting. Hrm...

I do see the convention seems to be not to have indent in these places:
https://github.com/openshift/openshift-ansible/blob/master/roles/etcd/templates/etcd.conf.j2#L3
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_openstack/templates/user_data.j2#L21

Looking closer at that last one, compare the current style:
```
{% if openshift_openstack_provision_user_commands %}
  - path: /root/ansible_install.sh
    permissions: '0544'
    content: |
{% for cmd in openshift_openstack_provision_user_commands %}
{% if cmd is string %}
      {{ cmd }}
{% elif cmd is iterable %}
      {{ cmd|join(' ') }}
{% endif %}
{% endfor %}
{% endif %}
```
vs including indent within the braces:
```
{% if openshift_openstack_provision_user_commands %}
  - path: /root/ansible_install.sh
    permissions: '0544'
    content: |
{%   for cmd in openshift_openstack_provision_user_commands %}
{%     if cmd is string %}
      {{ cmd }}
{%     elif cmd is iterable %}
      {{ cmd|join(' ') }}
{%     endif %}
{%   endfor %}
{% endif %}
```
I would argue that the extra indent improves code readability substantially... but just judged on consistency, yeah, I see the extra spaces are not consistent.